src/oEmbed.php

```diff
@@ -8,6 +8,8 @@
 
 /**
  * Polyfill for older WP versions.
+ *
+ * phpcs:disable PEAR.NamingConventions.ValidClassName.StartWithCapital -- this is purposefully using a lowercase letter for back compat reasons.
```

Comment:
This can safely be changed to `OEmbed` without breaking BC as class names in PHP are case-**in**sensitive.

Reply:
Perhaps I worded the note here incorrectly and I could have cleared that up with a better explanation.

The reason I chose not to rename here is less about BC and more a matter of my personal preference to encourage use of classnames with case sensitivity. I know that they are case in-sensitive by language convention but I just like to use them with matching cases.

Of course that's just me being pedantic. I will rename this and remove the ignore flag :)